control-plane/variables.tf

```diff
@@ -136,4 +207,10 @@ variable "r53_domain" {
   type        = string
   default     = ""
   description = "DNS domain for Route53 zone (defaults to domain if unset)"
-}
\ No newline at end of file
+}
+
+variable "sensitive_token" {
+  type = bool
+  default = true
+  description = "Boolean that determines if the module should treat the generated Rancher Admin API Token as sensitive in the output."
+}
```

Comment:
What is the benefit of making this configurable?

Reply:
Same reasoning as the change to `module.k3s.rancher_token` in `control-plane/outputs.tf`. See my comment on that change for details.